src/components/Restoration/ExtendTimeLimit.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <v-card flat id="extend-time-limit" class="pt-2 mr-8">
+
+    <!-- Extension Time Section -->
+    <section
+      id="extension-time-section"
+      class="section-container"
+      :class="{ 'invalid-section': invalidSectionExpiry }"
+    >
+      <v-row no-gutters>
+        <v-col cols="12" sm="3">
+          <strong :class="{ 'error-text': invalidSectionExpiry }">Extension Time</strong>
+        </v-col>
+        <v-col cols="12" sm="9">
+          <!-- Limited Restoration Radio Panel -->
+          <LimitedRestorationPanel
+            :currentDate="getCurrentDate"
+            :expiryDate="expiry"
+            :key="expiry"
+            :maxNumberOfMonths=36
+            @expiry="setRestorationExpiry(addMonthsToDate(previousNumberOfMonths, $event))"
+            @valid="setExpiryValid($event)"
+          />
+        </v-col>
+      </v-row>
+    </section>
+
+    <!-- Approval Type Section (if applicable) -->
+    <section id="approval-type-section"
+      class="section-container"
+      :class="{ 'invalid-section': invalidSectionApprovalType }"
+    >
+      <template v-if="approvalType == ApprovalTypes.VIA_COURT_ORDER">
```

Comment:
Only show if the previously filed limited restoration was approved by court order.